fairlearn/reductions/_moments/conditional_selection_rate.py

```diff
@@ -173,8 +184,52 @@ class EqualizedOdds(ConditionalSelectionRate):
 
     short_name = "EqualizedOdds"
 
+    def __init__(self, ratio=1.0):
+        super(EqualizedOdds, self).__init__()
+        self.ratio = ratio
+
+    def load_data(self, X, y, **kwargs):
+        """Load the specified data into the object."""
+        super().load_data(X, y,
+                          event=pd.Series(y).apply(lambda y: _LABEL + "=" + str(y)),
+                          **kwargs)
+
+
+class ErrorRatio(ConditionalSelectionRate):
+    r"""Implementation of Error Ratio as a moment.
+
+    Measures the ratio in errors between unprivileged and privileged attributes, i.e.
+    2 sided version of error ratio -
+    .. math::
+    1/r <= error(unpriv) / error(priv) <= r
+
+    This implementation of :class:`ConditionalSelectionRate` defines
+    events corresponding to the unique values of the `Y` array.
+
+    The `prob_event` :class:`pandas:pandas.DataFrame` will record the
+    fraction of the samples corresponding to each unique value in
+    the `Y` array.
+
+    The `index` MultiIndex will have a number of entries equal to
+    the number of unique values for the sensitive feature, multiplied by
+    the number of unique values of the `Y` array, multiplied by two (for
+    the Lagrange multipliers for positive and negative constraints).
+
+    With these definitions, the :math:`signed_weights` method
```

Comment:
Might want to expand on this section a bit, since the ratio approach isn't in the paper.

Reply:
I'll add some documentation which explains the ratio approach.

Reply:
I believe you wanted to revise this once more to remove the privileged/unprivileged formulation in favor of the "single sensitive feature vs. all" approach, so I'll hold off on reviewing the documentation. Thanks for putting in the work to write the doc, though! This will come in handy.

Reply:
I'll update the documentation and get back.